bazel/canbench.sh

```diff
@@ -43,8 +43,12 @@ elif [ "$1" = "--update" ]; then
         cp "${RUNFILES}/canbench_results.yml" "${REPO_RESULTS_PATH}"
     fi
 else
+    echo "CANBENCH_BIN: "${CANBENCH_BIN}
+    echo "POCKET_IC_BIN: "${POCKET_IC_BIN}
+    echo "pocket ic version:"
+    ${POCKET_IC_BIN} --version
     # Runs the benchmark test that fails if the diffs are new or above the threshold.
-    ${CANBENCH_BIN} >$CANBENCH_OUTPUT
+    ${CANBENCH_BIN} --no-runtime-integrity-check --runtime-path ${POCKET_IC_BIN} >$CANBENCH_OUTPUT
```

Comment:
nit: I *strongly* recommend keeping the integrity check. This means that you have to specify the version of Pocket IC that canbench expects (which is version 5.0). canbench isn't guaranteed to work with any other version, and using Pocket IC from HEAD will make these tests very flaky.

Reply:
canbench downloads pocketic v5.0 from [here](https://github.com/dfinity/pocketic/releases/tag/5.0.0).

Getting back to @basvandijk's original concern:

> is there such a tight coupling between canbench and pocket-ic that you need to match on hashes? Or would a minimal version check be good enough. If the latter is possible we could provide the HEAD version of pocket-ic as a Bazel dep in the above Bazel rule.

I do think it's important to provide exactly the version of pocketic that canbench expects for determinism and for avoiding flakiness.